client/components/chart/bar.jsx

```diff
@@ -1,15 +1,17 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' ),
-	debug = require( 'debug' )( 'calypso:module-chart:bar' );
+import React from 'react';
+import classNames from 'classnames';
+import debugModule from 'debug';
+
+const debug = debugModule( 'calypso:module-chart:bar' );
```

Comment:
This is correct, but let's move this below the `Internal dependencies` block and add a module variables block:

```javascript
/**
 * Module variables
 */
const debug = debugModule( 'calypso:module-chart:bar' );
```